package-lock.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "node-red-contrib-rhea",
+  "version": "0.0.3",
+  "lockfileVersion": 1,
+  "requires": true,
+  "dependencies": {
+    "debug": {
+      "version": "3.1.0",
+      "resolved": "https://registry.npmjs.org/debug/-/debug-3.1.0.tgz",
+      "integrity": "sha512-OX8XqP7/1a9cqkxYw2yXss15f26NKWBpDXQd0/uK/KPqdQhxbPa994hnzjcE2VqQpDslf55723cKPUOGSmMY3g==",
+      "requires": {
+        "ms": "2.0.0"
+      }
+    },
+    "ms": {
+      "version": "2.0.0",
+      "resolved": "https://registry.npmjs.org/ms/-/ms-2.0.0.tgz",
+      "integrity": "sha1-VgiurfwAvmwpAd9fmGF4jeDVl8g="
+    },
+    "rhea": {
+      "version": "0.2.12",
+      "resolved": "https://registry.npmjs.org/rhea/-/rhea-0.2.12.tgz",
+      "integrity": "sha1-laSDZ/pUxrBCBz58riE8EU5FT+A=",
+      "requires": {
+        "debug": "3.1.0"
+      }
+    }
+  }
+}
```

Comment:
not quite sure ... is this file needed? it wasn't on the remote repo before this PR.

Reply:
It is a best pratice from npm help : 
https://github.com/npm/npm/blob/v5.0.0/doc/files/package-lock.json.md